NanoAnalysis/python/ZZExtraFiller.py

```diff
@@ -36,6 +36,15 @@ def bookExtra(self, collName) :
         if self.isMC:
             self.out.branch(collName+"_dataMCWeight", "F", lenVar=theLenVar, title="data/MC efficiency correction weight", limitedPrecision=12)
 
+        # Book MELA angle branches
+        self.out.branch(collName + "_helcosthetaZ1", "F", lenVar=theLenVar)
+        self.out.branch(collName + "_helcosthetaZ2", "F", lenVar=theLenVar)
+        self.out.branch(collName + "_helPhi", "F", lenVar=theLenVar)
+        self.out.branch(collName + "_costhetastar", "F", lenVar=theLenVar)
+        self.out.branch(collName + "_phistarZ1", "F", lenVar=theLenVar)
+        self.out.branch(collName + "_mZ1", "F", lenVar=theLenVar)
```

Comment:
We already have the masses stored as _Z1mass, Z2mass; these should not be needed.